mne/epochs.py

```diff
@@ -2145,6 +2146,14 @@ def read_epochs(fname, proj=True, add_eeg_ref=True, verbose=None):
     verbose : bool, str, int, or None
         If not None, override default verbose level (see mne.verbose).
         Defaults to raw.verbose.
+    baseline : None or tuple of length 2 (default (None, 0))
+        The time interval to apply baseline correction.
+        If None do not apply it. If baseline is (a, b)
```

Comment:
This changes what `baseline=None` means from elsewhere a bit. I think I'd rather have `baseline='original'`, and if someone passes `baseline=None` it throws an error if epochs had been saved baseline-corrected. Otherwise someone might assume "if I load with baseline=None, my data will not be baseline corrected" which is not necessarily correct.